classes/ProductAssembler.php

```diff
@@ -92,27 +144,24 @@ private function addMissingProductFields(array $rawProduct): array
         if ($isStockSharingBetweenShopGroupEnabled) {
             $sql .= "  ON sa.id_product = p.id_product
 			        AND sa.id_shop = 0
+                    AND sa.id_product_attribute = 0
```

Comment:
Won't this display the wrong stock for product combinations?

Reply:
As I specified in PR description, there is no `id_product_attribute` entering these functions.

Originally, when this query was called one by one, it was randomly selecting entries from `ps_stock_available`, because the LEFT JOIN generated duplicate entries. This was not visible due to `LIMIT 1` originally present there.

Proper quantity is later retrieved in `Product::getProductProperties`, that's why it does not matter at this point.